components/LinkGroup/src/index.scss

```diff
@@ -1,15 +1,22 @@
 .denhaag-link-group__image {
   padding-block-end: var(--denhaag-link-group-image-padding-block-end);
+  width: var(--denhaag-link-group-image-size);
+  height: var(--denhaag-link-group-image-size);
```

Comment:
The aspect-ratio is always 1 by 1 ?

Reply:
@paulAcato I just checked with Joni, it is 140x140 pixels always aspect ratio 1:1.

Reply:
Check 👍